dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -110,25 +107,115 @@ def test_load_data_labeler(self, *mocks):
+        one_dim_pred = np.array(one_dim)
+        # for fit
+        self.assertTrue(
+            np.array_equal(np.array(one_dim),
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           one_dim, fit_or_predict='fit')))
+        self.assertTrue(
+            np.array_equal(np.array(one_dim),
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           pd.Series(one_dim), fit_or_predict='fit')))
+        self.assertTrue(
+            np.array_equal(np.array(one_dim),
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           np.array(one_dim), fit_or_predict='fit')))
+        # for predict
+        self.assertTrue(
+            np.array_equal(one_dim_pred,
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           one_dim, fit_or_predict='predict')))
+        self.assertTrue(
+            np.array_equal(one_dim_pred,
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           pd.DataFrame(one_dim), fit_or_predict='predict')))
+        self.assertTrue(
+            np.array_equal(one_dim_pred,
+                           BaseDataLabeler._check_and_return_valid_data_format(
+                           np.array(one_dim), fit_or_predict='predict')))
+
+        # test proper conversion of unstructured labels
+        labels = [[(0, 4, "BACKGROUND"), (4, 10, "ADDRESS")],
+                  [(0, 5, "SSN"), (5, 8, "BACKGROUND")]]
+        validated_labels = \
+            BaseDataLabeler._check_and_return_valid_data_format(labels)
+        self.assertIsInstance(validated_labels, np.ndarray)
+        self.assertEqual(len(validated_labels), 2)
+        self.assertEqual(len(validated_labels[0]), 2)
+        self.assertEqual(len(validated_labels[0][0]), 3)
+        self.assertEqual(validated_labels[0][0][0], 0)
+        self.assertEqual(validated_labels[0][1][1], 10)
+        self.assertEqual(validated_labels[1][0][2], "SSN")
+
+        # test proper conversion of data reader objects
+        for dt in ["csv", "json", "parquet"]:
+            data_obj = dp.Data(data=pd.DataFrame(two_dim), data_type=dt)
+            val = BaseDataLabeler._check_and_return_valid_data_format(data_obj)
+            self.assertTrue(np.array_equal(np.array(two_dim), val))
+
 
 label_encoding = {
-  "encoder": {
-    'PAD': 0,
-    'CITY': 1,  # SAME AS BACKGROUND
-    'BACKGROUND': 1,
-    'ADDRESS': 2,
-    'PERSON': 3,
-  },
-  "decoder": {
-    0: "PAD",
-    1: "BACKGROUND",
-    2: "ADDRESS",
-    3: "PERSON"
-  },
-  "count": 4
+    "encoder": {
+        'PAD': 0,
+        'CITY': 1,  # SAME AS BACKGROUND
+        'BACKGROUND': 1,
+        'ADDRESS': 2,
+        'PERSON': 3,
+    },
+    "decoder": {
+        0: "PAD",
+        1: "BACKGROUND",
+        2: "ADDRESS",
+        3: "PERSON"
+    },
+    "count": 4
```

Comment:
no need to change this

Reply:
I actually don't know how this happened, I'll revert it though